be/src/runtime/runtime_state.h

```diff
@@ -86,6 +86,8 @@ class RuntimeState {
     }
     int max_errors() const { return _query_options.max_errors; }
     int query_timeout() const { return _query_options.query_timeout; }
+    int insert_timeout() const { return _query_options.insert_timeout; }
```

Comment:
seems not used?

Reply:
Yes, I made this redundance on purpose as an access to insert_timeout val just for future use. If we need to remove it, then the query_timeout() method can also be removed because it's redundant too. Should I remove them then?

Reply:
Great. Just keep it.